File: PetBuddies-API/PetBuddies-API/Dtos/Consulta/ConsultaDto.cs
namespace PetBuddies_API.Dtos.Consulta
{
    public class ConsultaDto
    {
        public int Id { get; init; }

        public string TipoConsulta { get; init; } = string.Empty;

        public DateTime DataHora { get; init; }

        public string Status { get; init; } = string.Empty;

        public int AnimalId { get; init; }
    }
}

File: petbuddies-ai/src/main/java/br/com/fiap/petbuddies/dto/client/ConsultaDto.java
package br.com.fiap.petbuddies.dto.client;

import java.time.LocalDateTime;

public class ConsultaDto {

    private Long id;
    private String tipoConsulta;
    private LocalDateTime dataHora;
    private String status;
    private Long animalId;

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public String getTipoConsulta() { return tipoConsulta; }
    public void setTipoConsulta(String tipoConsulta) { this.tipoConsulta = tipoConsulta; }
    public LocalDateTime getDataHora() { return dataHora; }
    public void setDataHora(LocalDateTime dataHora) { this.dataHora = dataHora; }
    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }
    public Long getAnimalId() { return animalId; }
    public void setAnimalId(Long animalId) { this.animalId = animalId; }
}
